### PR TITLE
Add more information on packages

### PR DESCRIPTION
**Pull request by AMDmi3**
As per discussion in https://github.com/jaagr/polybar/commit/802e69abafa575e3e38d36cca0c1ce07028dfb2e

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -58,12 +58,20 @@ Some of the services included so far:
 
 ## Getting started
 
+<a href="https://repology.org/metapackage/polybar">
```
Comment (jaagr) [inline README.md]: Markdown has tags for this, e.g:

~~~ md
[![Packaging status](https://repology.org/badge/vertical-allrepos/polybar.svg)](https://repology.org/metapackage/polybar)
~~~

Comment (AMDmi3) [inline README.md]: There's a purpose for using HTML here: only with HTML it's possible to use ```align="right"``` attribute which makes the badge right-aligned and floating, so it doesn't create a gap in text, [see how it looks](https://github.com/AMDmi3/polybar/tree/more-packages#getting-started), which IMO is the best way of embedding it. If you don't like it I can turn it into markdown (thus make it non floating), a plain text link, or remove it completely.

Comment (jaagr) [inline README.md]: Oh I missed that part. Totally agree with you.

**Comment by jaagr** [comment]
Thanks

**Pull request merged**